examples/quick_start/scripts/inference/README.md

```diff
@@ -0,0 +1,73 @@
+# Deploying a Ranking model on Triton Inference Server
+The last step of ML pipeline is to deploy the trained model into production. For this purpose we use NVIDIA Triton Inference Server](https://github.com/triton-inference-server/server), which is an open-source inference serving software, standardizes AI model deployment and execution and delivers fast and scalable AI in production. 
+
+[Merlin Systems](https://github.com/NVIDIA-Merlin/systems/tree/main) library is designed for building pipelines to generate recommendations. Deploying pipelines on Triton is one part of the library's functionality and Merlin Systems provides easy to use APIs to be able to export ensemble graph and model artifacts so that they can be loaded on Triton with less effort.
+
+In this example we demonstrate the necessary steps to deploy a model to Triton and test it:
+
+1. Creating the ensemble graph
+2. Launching the Triton Inference Server
+3. Sending request to Server and receiving the response
+
+## Creating the Ensemble Graph
+
+In order to do model deployment stage, you are required to complete `preprocessing` and `ranking` steps already.  At the inference step, we do have a collection of multiple (individual) models to be deployed on Triton. We deploy NVTabular model as well to be able to transform raw data as we do in the preprocessing phase. In this context, deploying multiple models is called an ensemble model since it represents a pipeline of one or more models and the connection of input and output tensors between those models. Ensemble models are intended to be used to encapsulate a procedure that involves multiple models, such as "data preprocessing -> inference -> data postprocessing". 
```

Comment:
```suggestion
In order to do model deployment stage, you are required to complete `preprocessing` and `ranking` steps already from the [Quick-start for Ranking](../ranking.md).  At the inference step, we might have a collection of multiple (individual) models to be deployed on Triton. In this example, we deploy our NVTabular workflow model to be able to transform raw data the same way as in the dataset preprocessing phase, in order to avoid the training-serving skew. 

In this context, deploying multiple models is called an ensemble model since it represents a pipeline of one or more models that are sequentially connected, i.e., output of a model is the input of next model. Ensemble models are intended to be used to encapsulate a procedure that involves multiple models, such as "data preprocessing -> inference -> data postprocessing". 
```